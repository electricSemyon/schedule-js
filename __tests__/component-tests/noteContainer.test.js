import React from 'react';
import renderer from 'react-test-renderer';

import NotesContainer from '../../src/components/noteContainer.js'

import Fixture from '../fixtures/localStorageFixture.json'

test('renders notes from "notes" prop', () => {
  const notes = Fixture.data;
  const day = 1;

  const component = renderer.create(
    <NotesContainer notes={notes}
          day={day}/>
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
})
