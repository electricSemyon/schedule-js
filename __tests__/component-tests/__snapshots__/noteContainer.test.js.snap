exports[`test renders notes from "notes" prop 1`] = `
<div
  className="shedule">
  <div
    className="note ">
    <div
      className="id">
      Sun
    </div>
    <div
      className="body">
      react + redux
    </div>
  </div>
  <div
    className="note active">
    <div
      className="id">
      Mon
    </div>
    <div
      className="body">
      buming
    </div>
  </div>
  <div
    className="note ">
    <div
      className="id">
      Tue
    </div>
    <div
      className="body">
      buming
    </div>
  </div>
  <div
    className="note ">
    <div
      className="id">
      Wed
    </div>
    <div
      className="body">
      react + redux
    </div>
  </div>
  <div
    className="note ">
    <div
      className="id">
      Thu
    </div>
    <div
      className="body">
      buming
    </div>
  </div>
  <div
    className="note ">
    <div
      className="id">
      Fr
    </div>
    <div
      className="body">
      buming
    </div>
  </div>
  <div
    className="note ">
    <div
      className="id">
      Sat
    </div>
    <div
      className="body">
      html & css
    </div>
  </div>
</div>
`;
